### ISSUE TITLE
Share your personality type result in the app in KnowU Personality Test App.

### ISSUE THREAD

**Issue by bhask4r** [MSCKIIT/MSCxHacktoberfest#227]
It would be great to share your results in the app itself to enable rest of the users to check and see other people having same results as themselves.

I would like the usage of firebase to make this feature if possible.

Any other database solution would work like MongoDB, SQl, Django whatever is best for flutter apps.

It should be like getting your results and sharing it to the world after submitting your name, age and gender so need to have the UI for that as well.

Let me know if you want to make this feature a success!

**Comment by mk1107** [comment]
Hi, can you describe in detail what we need to do? I have knowledge about flutter and willing to work

**Comment by arnab8335** [comment]
@bhask4r explain him what needs to be done and kindly assign him.

**Comment by bhask4r** [comment]
@mk1107 Hello, See the idea is, when a user give the test and is able to see his result let us say the result was "INTJ" and all other details about INTJ is shown on the screen.
You can see there is a button at the bottom above which a text says "Share your results on our page." so that is basically a feature I wanted to add but still haven't yet, so if you understood what needs to be done let me know and I will assign you the task.
Again I just want a feature to share our result on another screen which can be seen by other users that is it.

**Comment by mk1107** [comment]
Yea I understood that but for it I would require a database where things will be uploaded
For that I would need some access to firebase or so ..

**Comment by bhask4r** [comment]
> Yea I understood that but for it I would require a database where things will be uploaded For that I would need some access to firebase or so ..

Yeah sure if you want to work with firebase I can give you the access, tell me your email address.

**Comment by mk1107** [comment]
So u have storage in your firbase??

**Comment by bhask4r** [comment]
> So u have storage in your firbase?? I guess that's paid part and I know how to do like I can tell You but not having that much time to implement.

isn't storing strings part is free?

**Comment by mk1107** [comment]
u need to create a database so it is going to store which will require space so..

**Comment by innatepanda** [comment]
@mk1107 Upto a certain limit, storing in firebase is free for all.

**Comment by bhask4r** [comment]
> @mk1107 Upto a certain limit, storing in firebase is free for all.

Yes exactly.
